api/v1/zz_generated.deepcopy.go

```diff
@@ -113,12 +113,27 @@ func (in *DpuOperatorConfigStatus) DeepCopy() *DpuOperatorConfigStatus {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
Did you try building with these changes to deepcopy? It fails for me.

Reply:
We need to fix the discrepancy between your env and mine in a separate PR before we tackle this new PR.

Reply:
removing this change manually from the commit for now. We will bump to 1.22.3 early next week.